### PR TITLE
Added color for test response

### PR DESCRIPTION
**Pull request by GatewayBit**
Little extra color to the output. Looks good on light and dark terminal color schemes!

### CODE REVIEW

File: twa [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 # twa: a tiny website auditing script
 
+
+# Colored output
+GREEN='\033[0;32m'
```
Comment (woodruffw) [inline twa]: Prefix these with `COLOR_`, e.g. `COLOR_BLUE` and `COLOR_RESET` (`NC` isn't accurate; sending `\033[0m` will cause all current attributes to reset. There's `\033[39;49m`, but fewer terminals support it).

**Comment by woodruffw** [comment]
Two nits, but otherwise this looks good. Thanks for adding it!

**Comment by GatewayBit** [comment]
Thanks for the constructive feedback! I've added the recommended changes in the latest commit.

**Comment by woodruffw** [comment]
Thank you! I'll merge this tonight.

**Comment by woodruffw** [comment]
Merged, thanks again!

**Pull request merged**